vendor/bat-native-ledger/src/bat/ledger/internal/gemini/gemini_authorization.cc

```diff
@@ -197,8 +201,6 @@ void GeminiAuthorization::OnPostAccount(
 
   wallet_ptr->token = token;
   wallet_ptr->user_name = name;
-  wallet_ptr->status =
```

Comment:
Do we need to set status to `VERIFIED` here?

Reply:
We set the status to `VERIFIED` after wallet linking succeeds - https://github.com/brave/brave-core/blob/master/vendor/bat-native-ledger/src/bat/ledger/internal/gemini/gemini_authorization.cc#L241

Reply:
Awesome - thanks!